js/core/component.js

```diff
@@ -381,4 +381,11 @@ const Component = Class.inherit({
     }
 });
 
-export default Component;
```

Comment:
Should we announce a breaking change for this?

Reply:
I guess no, `Component` is not a public class, no user should use it